client/Component/app.jsx

```diff
@@ -20,11 +20,16 @@ class App extends React.Component {
     } else {
       endpoint = window.location.pathname.split('/')[1];
     }
-    fetchData(endpoint);
+    axios.get(`/airbnb/listings/${endpoint}`)
+      .then((data) => {
+        this.setState({
+          photos: data.data,
+        }, () => {
+          console.log('CURRENT STATE', this.state);
+        });
+      });
   }
```

Comment:
you might be able to use supertest for this. I was reading about this last night. Basically you can mock a fetch request without having to actually start your server. I will also try to refactor mine to use super test. I am not too clear on the details of it just yet.

Reply:
Thank you for the details, I was trying to mock and Axios request using __mock__ but it was not functioning as expected. That may be another option. I will explore supertest.